apps/site/src/lib/admin/useApplicant.ts

```diff
@@ -0,0 +1,135 @@
+import axios from "axios";
+import useSWR from "swr";
+
+import { ParticipantRole, Status, Uid, Score } from "@/lib/userRecord";
+
+export type Review = [string, Uid, Score];
+
+// The application responses submitted by an applicant
+export interface HackerApplicationData {
+	email: string;
+	pronouns: string[];
+	ethnicity: string;
+	is_18_older: boolean;
+	school: string;
+	education_level: string;
```

Comment:
Thought: since interfaces can be extended, we could combine some of the shared fields into a common interface to reduce the duplication.